src/ui/qgscodeditorsettings.ui

```diff
@@ -42,7 +42,7 @@
        <item>
         <widget class="QStackedWidget" name="mPreviewStackedWidget">
          <property name="currentIndex">
-          <number>4</number>
+          <number>0</number>
```

Comment:
Was that deliberate, or something QtCreator accidentally changed? Is the new value correct (or are we just overriding this in the cpp? :))

Reply:
It's overriden, but seems nicer to default to first page in the ui file